docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,90 @@
+
+Alone, a CSM is of little practical use. The standard defines an interface but does not define where necessary sensor metadata should be sourced, the format of said metadata, or the implementation of the photogrammetric algorithms used by the sensor. Therefore, a CSM exists within a broader ecosystem. First, some external capability or method must be used to generate image support data (ISD) that can be used to parameterize a CSM. Then, the CSM can parse and use the ISD to extract the parameters (interior and exterior orientation) necessary to perform photogrammetric operations (as defined by the CSM API). Finally, a SET is likely used to orchestrate the pieces of the CSM ecosystem and exploits said pieces in order to use the sensor for some desired operation. (@TODO citation)
+
+!!! INFO "CSM -- Quick Definition"
+    The CSM is a standard that defines an API for sensor models, thereby providing a common structure that guarantees interoperability of any sensor models that adhere to the standard.
+
+## The ASC Sensor Model Ecosystem
+The Astrogeology Science Center has created and maintains a suite of software packages that provide end-to-end processing for sensor models, including a metadata specification, CSM-compliant sensor model implementation, and tools to leverage those sensor models for scientific processing.
+
+### USGS Community Sensor Model (USGSCSM)
+The USGS Community Sensor Model is a concrete implementation of sensor models according to the standards described in the CSM.  Where the CSM is the set of rules and standards that guides the creation of sensor models, the USGSCSM is a library of sensor models that adheres to those rules.  The USGSCSM library provides generic instances of sensor models for instantaneous (framing) cameras, time-dependent (line-scan) cameras, push-frame cameras, synthetic aperature radar (SAR), and projected sensor models.  Additionally, USGSCSM provides an extensible plugin architecture that allows for additional, interface-compliant sensor models to be dynamically loaded.
+
+@TODO Introduce ISDs
+
+Sensor models implemented within USGSCSM are stateful in the sense that their underlying geometries can be modified.  Moreover, it is possible to save a model's current state to a _state string_ so that a future model can be instantiated with that exact model state.  This is an important capability when performing incremental modifications via a process like bundle adjustment, particularly if the user decides to undo those adjustments or share the modified state with collaborators.
+
+@TODO is it necessary to discuss reasons for modifying geometries?
```

Comment:
I think so, it worth pointing out that the accuracy of these models is always changing.

Reply:
Not in here though?

Reply:
I don't think it's a big deal one way or another. CSM states are a lot easier to modify compared to ISIS camera which is a big advantage they have. Mentioning that briefly I don't think is a big deal as I feel this is a more of a summary. 

If you're thinking more about a break down of internal and external orientations and how they all affect the camera model and why you would want to change those, then yeah, that is something that can be written later and referenced here.